src/components/modules/admin/pages/AdminMonthlyReviewEditorPage.tsx

```
@@ -0,0 +1,447 @@
+              </div>
+              <div className="p-3 border rounded-xl bg-card">
+                <span className="text-xs text-muted-foreground uppercase font-mono block">Smart Contracts</span>
+                <span className="text-lg font-bold text-foreground">
+                  {onChainMetrics.soroban.invocationCount} calls
+                </span>
+              </div>
+            </div>
+          </div>
+        ) : (
+          <p className="text-sm text-muted-foreground py-4 text-center border border-dashed rounded-xl">
+            No on-chain metrics loaded yet. Select a network and click 'Refresh metrics' to pull real-time data!
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)